src/main/kotlin/uk/gov/gdx/datashare/repository/EventData.kt

```diff
@@ -9,13 +9,12 @@ import java.time.LocalDateTime
 data class EventData(
   @Id
   val eventId: String,
-  val eventType: String,
-  val dataProvider: String,
-  val datasetType: String,
+  val eventTypeId: String,
```

Comment:
Could we use something like
```@Enumerated(EnumType.STRING)```
to shift the mapping to strings down to the framework, rather than something we need to worry about directly?

Reply:
This isn't JPA so can't use this at the moment.  Reactive does work with JPA - However we can change this in future